tests/instrumentation-framework/src/test/resources/testOpenTracingPolyglot/test.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "OpenTracing polyglot test",
+  "environment": {
+    "dockerCompose": "docker-compose.yml",
+    "initWaitSeconds": "20",
+    "type": "APM",
+    "apmAddress": "172.16.158.1"
+  },
+  "tests": [
+    {
+      "description" : "Docker compose JBoss Ticker Monster",
+      "skip": false,
+      "action": "action.sh",
+      "scriptServiceName": "nodejs",
+      "afterActionWaitSeconds": "20",
+      "verify": {
+        "jsonPath": [
+          {
+            "path": "$.length()",
+            "result": 1
```

Comment:
I have used `ConsoleRecorder` in node.js app for a reason. Do you remember your chat on IRC?  This node.js app depends on released  version of JS provider, therefore if we do some changes to the model then the app would fail to report traces to the server and if we expect two trace objects then test would fail. 

As you said the goal of this test is to test context propagation.